src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,37 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  let commentsExists;
+
+  if (post.comments !== undefined && post.comments.length > 0) {
+    commentsExists = true;
+  }
```

Comment:
Using let in the global scope of components isn't a good idea for code support and readability. 
Try to make another validation without let and using type in the condition.